wbType,wbLevel,wbParentType,wbPath,wbHandleError,wbStatus,wbFinalExec,wbExceptionType,wbStart,wbParentStart,wbDurationSec,wbDurationHrs,message,level,wbKey,configError,optionalInfo
UiPEngineLocal,0,,UiPEngineLocal,Rethrow,Failed,True,IOException,03/14/2018 10:21:42,03/14/2018 10:21:42,0.9375118,0.000260558833333333,Work Block UiPEngineLocal failed with exception The process cannot access the file 'C:\Users\mihai.dunareanu\Google Drive\Workspace\UiPath\Templates\UiPEngineLocalProcess\Data\Config.xlsx' because it is being used by another process. at Source Read range settings,Error,9c50c2c1e829220f4c3d131c911985b5,False,No info
